### ISSUE TITLE
llc-comms group doesn't appear in menus

### ISSUE THREAD

**Issue by ghwood** [ietf-tools/datatracker#7318]
### Describe the issue

I expect the llc-comms group to appear in the Groups->IETF LLC menu but it doesn't. 
<img width="1017" alt="image" src="https://github.com/ietf-tools/datatracker/assets/5347540/efe78b97-d29b-4feb-a70f-c37715c8cda2">

(I don't see it anywhere else in the menu structure either.)

Perhaps related: The usual pattern for LLC groups (e.g. https://datatracker.ietf.org/adm/llc-board/about/) also doesn't seem to work.

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by rjsparks** [comment]
https://datatracker.ietf.org/adm/llc-board/about/ works fine for me?
But I think you meant https://datatracker.ietf.org/adm/llc-comms/about/
It is a group of type `team` just like the tools-team so you can get the meetecho behavior you were looking for.
Moving it to be some other kind of group and retaining those behaviors will require changing how _all the other groups of that type_ behave.

Also, please try to forget that `/{grouptypeacronym}/{groupacronym}` exists. It will stop working in the future as soon as I find a better way to provide a URL to discuss all the groups of type grouptype. Use `/group/{groupacronym}` instead. Always. All the previous variants are, or will soon be, redirecting to that URL pattern.

**Comment by ghwood** [comment]
Yes, I meant `~/adm/llc-board/about` as an example that works whereas `~/adm/llc-comms/about` does not, and now I understand why it doesn't and won't in  the future.

But the main point is that I would expect llc-comms to appear in the menu `Groups->IETF LLC->,` but it doesn't. Are my expectations wrong here as well?

**Comment by rjsparks** [comment]
the groups menu is built based on the group type. the llc-comms group currently is built under "active teams".

**Comment by rjsparks** [comment]
To reiterate, and be a bit more explicit - llc-comms is currently at team because it behaves like a team. All groups of a given type behave the same way - it's controlled with a GroupFeature instance for the group's type: https://github.com/ietf-tools/datatracker/blob/280481b92492e48043ba4df9c7d0ef8958f04ffd/ietf/group/models.py#L238-L282

**Comment by ghwood** [comment]
Thank you for taking the time to provide the detail. I now understand the situation. Given the current model, I don't have a good suggestion about how to reconcile the desired group behavior with expected menu structure.

**Comment by rjsparks** [comment]
Adding the label to flag this for review as we go forward.
